src/features/import/hooks/useColumn.ts

```diff
@@ -53,7 +54,7 @@ export default function useColumn(orgId: number) {
         return column.field == value.slice(6);
       }
 
-      if (column.kind == ColumnKind.DATE) {
```

Comment:
This line of code stumped me as I was reading it. I couldn't understand why date and enum shared the same logic, and then I realized that it's just about string length.

Although `date` and `enum` happen to have the same length, they are not related in any way. Writing the logic this way makes it seem like they are, and that can be confusing to a reader.

I think there should be a separate if statement for `ColumnKind.ENUM`.

Reply:
Sure, fixed